api/portainer.go

```diff
@@ -373,6 +373,8 @@ type (
 		Name string `json:"Name" example:"my-environment"`
 		// Environment(Endpoint) environment(endpoint) type. 1 for a Docker environment(endpoint), 2 for an agent on Docker environment(endpoint) or 3 for an Azure environment(endpoint).
 		Type EndpointType `json:"Type" example:"1"`
+		// ContainerEngine represents the container engine type. 'docker' or 'podman' are valid values.
+		ContainerEngine string `json:"ContainerEngine" example:"docker"`
```

Comment:
Let's keep the string based property for flexibility but let's introduce constants in this file for supported values; something like:

```
const (
  ContainerRuntimeDocker = "docker"
  ContainerRuntimePodman = "podman"
)
```

Reply:
Updated